packages/core/strapi/lib/commands/opt-in-telemetry.js

```diff
@@ -53,12 +53,12 @@ const generateNewPackageJSON = (packageObj) => {
 
 const sendEvent = async (uuid) => {
   try {
-    await fetch('https://analytics.strapi.io/track', {
+    await fetch('https://analytics.strapi.io/api/v2/track', {
```

Comment:
Since this URL is used in multiple places: could we make it a constant?

Reply:
Yes, this is a good point. Where should we put it though, since url is present across a lot of packages?

Reply:
That's fine, but I'm not aware of where we keep any constants/configs of this sort in the monorepo. Could you point me towards it if you know? Or are you suggesting creating a space for it?